packages/components/src/components/Drawer/Drawer.test.tsx

```diff
@@ -48,14 +48,15 @@ describe("<Drawer />", () => {
 
   it("should open a drawer and close it by clicking an element in the background", async () => {
     render(<Default />);
+
     const renderedOpenButton = screen.getByText(OPEN_DRAWER_TEXT);
     expect(renderedOpenButton).toBeInTheDocument();
 
     await user.click(renderedOpenButton);
     expect(await screen.findByRole("dialog")).toBeInTheDocument();
     expect(screen.getByRole("heading", { level: 2 })).toBeInTheDocument();
 
-    await user.click(renderedOpenButton);
+    await user.click(await screen.findByRole("presentation"));
```

Comment:
Why the issue it's related to the place is clicked? The flakiness is also visible in the test to check the `Escape` key.

Reply:
we can remove the animations while running tests to check if that is also affecting it but firing a click on a button that should be hidden on the interface doesn't seem a good test

Reply:
You have a good point about this test. I didn't notice that this click is on the button that is not visible. So, for this test, it makes sense to click on the presentation.